### DISCUSSION TITLE
What triggers translations to be downloaded?

### DISCUSSION THREAD

**Discussion by Kolby-Udacity** [crowdin/github-action#123] [Q&A]
What triggers translations to be downloaded? Is a webhook utilized, or is something checking Crowdin on an interval for changes? Thanks

**Comment by Olena1234** [comment]
Hi there! Could you please share more details about your question and project name as well?

**Comment by andrii-bodnar** [comment]
Hi @Kolby-Udacity!

You can set up triggering by cron, or trigger download for pushes to some branches or even on pull requests only. 

Please read the [Events that trigger workflows](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows) for more details.
